lib/beacon/live_admin/live/page_editor_live/revisions.ex

```diff
@@ -79,7 +79,6 @@ defmodule Beacon.LiveAdmin.PageEditorLive.Revisions do
   ## FUNCTION COMPONENTS
 
   attr :event, :map
-  attr :latest, :boolean, default: false
```

Comment:
This is a leftover I forgot to remove, from before I found the `group` + `hidden group-first:block` solution for displaying the "Latest" badge via CSS